data/hai/unfettered side missions.txt

```diff
@@ -203,29 +139,55 @@ mission "Hai Legacy 1"
+				goto choice
 			label take
 			action
-				give ship "Ancient Ladybug" "Good Omen"
-				fine 6500000
-				log "Accepted a somewhat shady deal for an old Hai ship, a Ladybug, that seems to still be in good functioning state."
-			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go through standard means for this sale. We will instead present it as a fine. You'll be charged for carrying Korath tech you had no idea was forbidden, and it will not go on your permanent record."`
-			`	You quickly realize they are not asking you for your opinion at all. In fact, you receive notice of the fine before they have even finished speaking. "We will need to make some modifications to the anti-missile system of the ship before you can use it. I also strongly would advise against repainting it; you will understand why later. I would not bother removing the rust, either, if I were you, it is not worth the risk."`
-			`	"Oh, and I almost forgot: make sure you return to the spaceport to get the cargo loaded onto your ship! Naturally, we will expect you use it as your flagship."`
+				give ship "Anomalocaris" "Good Omen"
+				fine 10000000
+				log "Accepted a somewhat shady deal for an old Hai ship, an Anomalocaris, that seems to still be in good functioning state."
+			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go through standard means for this sale. We will instead present it as a fine. You'll be charged for carrying Korath tech you had no idea was forbidden, and it will not go on your permanent record. These Detainers are suspicious looking indeed."`
+			`	You quickly realize they are not asking you for your opinion at all. In fact, you receive notice of the fine before they have even finished speaking. "We will need to make some modifications to the ship before you can use it. I also strongly would advise against repainting it; you will understand why later."`
+			`	"Oh, and I almost forgot: make sure you return to the spaceport to get the cargo loaded onto your ship! Naturally, we will expect you use it as your flagship, and you should also be using it as your only ship to not scare the Unfettered and delay this urgent delivery any more than it should be."`
+			choice
+				`	(Go back to the shipyard.)`
+			`	They walk away far enough so that you cannot hear them, leaving you with the access codes to the ship so you can bring it to the shipyards. Surprisingly you manage to bring it all the way there without issues despite this ship being meant for more than one crew member, thanks to the aid of the navigation AI. Unfortunately it shuts down once you arrive back in the shipyard, forcing you to fill up its crew.`
+			`	You remember they asked you to use this ship as flagship, and only ship, so it may be time to sell your other ships, or you could always park them later.`
```

Comment:
```suggestion
			`	You remember the mysterious pair requested you bring only the Anomalocaris. It may be time to sell your other ships, or you could always park them for later.`
```